### PR TITLE
Fix #88 - Replace Gunicorn with uWSGI

### LINKED ISSUE #88: Replace Gunicorn w/ uWSGI
### Proposed Changes

Self-explanatory

### Justification

uWSGI is better for the common case and more stable. Our recommendation is to use uWSGi over Gunicorn in production, so we want to revise the requirements and the documentation to support this.

### PR DESCRIPTION
**Pull request by jathanism**
### Fixes: #88

This adds a `nautobot-server start` command that is just a thin veneer
over `pyuwsgi.run()`, itself being the Python module implementation of
uWSGI.

This also adds `django-webserver` as a dependency which provides the
management command internals that bind to `pyuwsgi`. This project is maintained by one of the uWSGI contributors, so it's pretty solid.

### CODE REVIEW

**Comment by jathanism** [comment]
This _addresses_ the issue with #95 by circumventing it entirely, but it does not _fix_ the underlying issue with WHY the plugins are not being loaded correctly. That will take more time, but this will alleviate the problem for the near term by launching uWSGI from within the server utility instead of the other way around, with Gunicorn/uWSGI invoking the application.

File: pyproject.toml [hunk 1]
```diff
@@ -64,7 +66,6 @@ GitPython = "~3.1.13"
 # GraphQL support
 graphene-django = "~2.15.0"
 # WSGI HTTP server
```
Comment (glennmatthews) [inline pyproject.toml:68]: Go ahead and remove this comment as well please. :-)

Comment (jathanism) [reply pyproject.toml:68]: Oh yeah `poetry remove gunicorn` left the comment.

File: contrib/uwsgi.ini [hunk 2]
```diff
@@ -0,0 +1,29 @@
+[uwsgi]
```
Comment (glennmatthews) [inline contrib/uwsgi.ini:1]: Going to remove this file?

**Review by glennmatthews — APPROVED** [review]
Docs look good, will take your word that this works well. :-)

File: contrib/uwsgi.ini (hunk 2 above)
Comment (jathanism) [reply contrib/uwsgi.ini:1]: Yes, going to remove the entire `contrib` directory and a smattering of other vestigial files all in another pass soon. Thanks! :)

**Pull request merged**